statbot/sql.py

```diff
@@ -972,6 +972,7 @@ def remove_member(self, trans, member):
         # Don't delete role membership
 
     def remove_old_members(self, trans, guild):
+        # pylint: disable=singleton-comparison
```

Comment:
Can you leave a note as to why this is here? I don't like seeing random pylint suppressions, haha

Reply:
It complains about the `self.tb_guild_membership.c.is_member == True`.

Reply:
Hm, I see. Are we really concerned that `self.tb_guild_membership.c.is_member` could return a non-bool?

Reply:
Well it's not a python `bool`, it's a weird SQLAlchemy column/type/schema something object. When you do `table.c.column == value`, it generates an "equality" object, which it uses to create the `WHERE` clause of the query.

Reply:
Can you amend your comment to include that? That is super instructive.